projects/packages/forms/src/dashboard/inbox/util.js

```diff
@@ -10,6 +10,15 @@ export const getDisplayName = response => {
 	return response.ip;
 };
 
+export const getMonthName = monthNumber => {
+	const date = new Date();
+	date.setMonth( monthNumber - 1 );
+
+	return date.toLocaleString( 'en-US', {
```

Comment:
I think you probably want to use [dateI18n](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-date/#datei18n) here or even as a replacement for this function providing compatibility for all locales.

Reply:
Thanks for pointing that out, @ice9js.
My latest commit aims to fix this.
Let me know if you thought of anything different or if I can improve anything else.